android/keyframes-sample/build.gradle

```diff
@@ -20,7 +20,6 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
any specific reason to remove this? I believe we currently depend on library jar being copied here to build the sample app against.